templates/demo/app/dapp.js

```diff
@@ -18,6 +18,7 @@ class App extends React.Component {
     this.handleSelect = this.handleSelect.bind(this);
 
     this.state = {
+      error: false,
```

Comment:
Can you put `null` or empty string. because with `false`, it looks like it's only a boolean value.

Reply:
Why does it need to be set then?

Reply:
Legit, you can not put it.
It's just cleaner to declare state defaults in the constructor.

Reply:
Good point.